sdk/cognitiveservices/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient_private.cs

```diff
@@ -0,0 +1,216 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Azure.AI.TextAnalytics
+{
+    public partial class TextAnalyticsClient
```

Comment:
This type is huge even with one file in it. Have we ever considered delegating some of its implementation to additional internal types?

Reply:
Btw, does this file has any non-static methods?

Reply:
#8711